posts/d3-tutorial-points-axes.md

```diff
@@ -0,0 +1,139 @@
+---
+authors:
+- Justin Risch
+tags:
+- d3.js
+- tutorial
+- data governance
+- data visualization
+date: 2017-08-17T18:22:50.000Z
+title: "Plotting Points on Two Axes with D3.JS"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2019/01/alignmentgraph.png
+---
+
+Welcome to this simple tutorial on how to plot a point on Two Axes, along with some nice formatting techniques. As well, we'll be using both linear scaling and string interpolation.
```

Comment:
I would add a line about D3 and possible link to their main page https://d3js.org/

Reply:
Great idea! Doing it.